src/graphql/mutation/createSavedSearch.graphql

```diff
@@ -0,0 +1,15 @@
+mutation createSavedSearch($name: String!, $filters: LoanSearchFiltersInput) {
+  my {
+    createSavedSearch (savedSearch :{
+      filters: $filters,
+      name: $name,
+      isAlert: false,
+      queryString: $name,
+      sortBy: popularity
+    })
```

Comment:
Can you please add and save the full URL into the `url` param?

Reply:
We use the `url` field to link back into lend/filter (or legacy lend) from the `/lend/saved-search` page.

Reply:
Nevermind. That's not a possible param!